camelot/view/action_steps/crud.py

```diff
@@ -42,24 +43,27 @@ def _to_dict(self):
                 "row": row,
                 "tool_tip": header_item.data(Qt.ItemDataRole.ToolTipRole),
                 "icon_name": header_item.data(Qt.ItemDataRole.WhatsThisRole),
-                "object": header_item.data(ObjectRole)
+                "object": header_item.data(ObjectRole),
+                "verbose_identifier": header_item.data(VerboseIdentifierRole),
+                "valid": header_item.data(ValidRole),
+                "message": header_item.data(ValidMessageRole)
+                # FIXME: more roles? see camelot/view/crud_action.py:127
             })
             for column, item in items:
                 cell_data = {
                     "row": row,
                     "column": column,
                 }
-                for role in range(Qt.ItemDataRole.DisplayRole, ChoicesRole+1):
+                for role in range(Qt.ItemDataRole.DisplayRole, EndRoles):
                     if role in non_serializable_roles:
                         continue
                     role_data = item.data(role)
                     if role_data is not None:
+                        if role == Qt.ItemDataRole.EditRole and isinstance(role_data, Entity):
```

Comment:
komt dit voor ??

Reply:
Ja, voor City en NatuurlijkPersoon. Bij makelaarsnetwerk als ik het mij goed herinner (of de tableview daarboven/onder in de navigation panel).